docs/mesos/PrepareExternalServices.md

```diff
@@ -0,0 +1,54 @@
+# Prepare External Services for Mesos + Calico Deployment
+*Most Mesos clusters will run these services outside of their core Mesos cluster, separate from the Masters and Agents which read from them.*
+
+## Docker
+
+- **Docker must be installed on each Mesos Agent** that is deploying calico via the packaged Calico container (recommended). Users who prefer to [run Calico as a baremetal service](#) do not need to install Docker on each Agent.
+- **Docker must be installed on Mesos Master** if etcd and ZooKeeper are being deployed as Docker containers. Users who are running etcd / ZooKeeper elsewhere do not need to install Docker on each Master.
+
+Run the following commands to install Docker:
```

Comment:
I need to think about this more, so don't jump on this yet, but -

It doesn't make much sense that we're instructing users on how to install docker on their own systems. Realistically we should just link them to the docker install information for centos. This step made sense within Spike's guide as he was covering every single step between bare metal and a running cluster. But in these guides, I'm not sure it makes as much sense. 

The only reason I'm cautious to remove it is that the official docker docs don't mention anything about `docker-selinux`, as its a centos maintained rpm. And I can't find any documentation from centos covering how to install it...